### DISCUSSION TITLE
How come clustering can't work with SQLite?

### DISCUSSION THREAD

**Discussion by nlferman** [quartznet/quartznet#1014] [Q&A] [answered]
When setting up ADO.Net Jobstore for SQLite with clustering set to true, I am getting the following error message:

Exception thrown while trying to start the scheduler: SQLite cannot be used as clustered mode due to locking problems

Is there any work around for this?

**Comment by lahma** [comment, marked as answer]
This behavior has been discussed [here](https://github.com/quartznet/quartznet/issues/845#issuecomment-661677303) .

**Comment by neyugnluan** [comment]
I don’t thing it work, bcoz sqlite is a database file, 
Not same with client server with multiple connection at the same time, that’s why can’t support clusters

Workaround can be create a queue group work, each group own different data source 
That mean you have a lot of SQLite file
